zenodo/modules/fixtures/data/licenses.json

```diff
@@ -0,0 +1,1451 @@
+[
```

Comment:
Doesn't many of these licenses already exist in the existing vocabulary from Invenio-OpenDefinition